pkg/stub/handler.go

```diff
@@ -304,6 +305,52 @@ func (h *Handler) updateCfgArch(cfg *v1.Config) *v1.Config {
 	return cfg
 }
 
+func (h *Handler) tbrInaccessible() bool {
+	if h.configclient == nil {
+		// unit test environment
+		return false
+	}
+	// we still check for this in case we have proxies in a ipv6 env;
+	// if proxies will not be supported in ipv6 single stack, we can
+	// remove this
+	if util.IPv6Only() {
+		return true
+	}
```

Comment:
Do we need to do this here? Consider the following deployment topology:

1. IPv6 cluster that has no direct connection to the internet.
2. Proxy set up in a dual-stack DMZ (allows IPv4 and IPv6 connections).
3. IPv6 cluster configured to be proxied via the DMZ.

In this case the samples is in an IPv6 cluster, but may nonetheless be able to connect to registry.redhat.io. This may be an edge case, and the work-around is simple if the customer wants samples content.

Reply:
aside from this, per the earlier discussion, proxy in ipv6 being supported also confirms removal per the code comment above

hostnames still leverage DNS (in addition to static config and SLAAC) in ipv6, so the hostname below should sort out regardless of ipv6 / ipv4

Net ... I'm removing :-)